python/docs/source/user_guide/index.rst

```diff
@@ -26,6 +26,7 @@ PySpark specific user guides are available here:
    :maxdepth: 2
 
    python_packaging
+   basic_dataframe_operations
```

Comment:
We should probably put this under `sql/index` (and probably rename from `Spark SQL` -> sth like `DataFrame and SQL`)

Reply:
Okay